### DISCUSSION TITLE
23.12.26 데일리 스크럼

### DISCUSSION THREAD

**Discussion by minju412** [petit-a-petit/MIML#96] [Daily Scrum]
.

**Comment by minju412** [comment]
- 오늘은 어떤 일을 했나요?
    - 발표 준비
- 내일은 어떤 일을 하나요?
    - 테스트 코드 수정 및 반영
    - nGrinder 설치 및 세팅

**Comment by Jeonwoojae** [comment]
- 오늘은 어떤 일을 했나요?
  - 발표 외에 작업을 못했습니다..
- 내일은 어떤 일을 하나요?
  - nGrinder 학습

**Reply by Jeonwoojae** [reply]
docker로 사용하기
